src/EnergyPlus/LowTempRadiantSystem.hh

```diff
@@ -592,17 +504,100 @@ namespace LowTempRadiantSystem {
                                   LowTempRadiantSystem::SystemType const SystemType // Type of radiant system: hydronic, constant flow, or electric
     );
 
-    void UpdateRadSysSourceValAvg(bool &LowTempRadSysOn); // .TRUE. if the radiant system has run this zone time step
+    void UpdateRadSysSourceValAvg(EnergyPlusData &state,
```

Comment:
And I was expecting this possibility as well.  Looks like just this one function needed a state argument though, so that's great!